Tutorials update (#136)
coreos/fedora-coreos-docs: modules/ROOT/pages/tutorial-containers.adoc

= Setting up SSH access and starting containers at boot

[NOTE]
====
Make sure that you have completed the steps described in the xref:tutorial-setup.adoc[initial setup page] before starting this tutorial.
====

In this tutorial, we will setup SSH access and start a container at boot. Fedora CoreOS is focused on running applications/services in containers thus we recommend trying to only run containers and not modifying the host directly. This makes automatic updates more reliable and allows for separation of concerns with the Fedora CoreOS team responsible for the OS and end-user operators/sysadmins responsible for the applications.

> As usual we will setup console autologin, an hostname and disable kernel console login, but we will also:

Reviewer: ```suggestion
As usual we will setup console autologin, a hostname, and raise kernel logging level, but we will also:
```
